Patches/CP Metal Gear Solid/CP_MGS_CE_Patch_MeleeWeapon.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Patch>
+	<Operation Class="PatchOperationSequence">
+		<success>Always</success>
+		<operations>
+
+			<li Class="CombatExtended.PatchOperationFindMod">
+				<modName>[CP] Metal Gear Solid</modName>
+			</li>
+
+			<!-- ========== Longsword / Katana ========== -->
+
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="RHMelee_MGS_HFBlade"]/tools</xpath>
+				<value>
+					<tools>
+						<li Class="CombatExtended.ToolCE">
```

Comment:
I'm alright with it being powerful, but I still might tone it down a ways--reference the Wraithblade from Combat Extended: Melee below. Additionally, you will probably want to add <chanceFactor> tags to the attacks as appropriate.
``` <tools>
      <li Class="CombatExtended.ToolCE">
        <label>handle</label>
        <capacities>
          <li>Blunt</li>
        </capacities>
        <power>2</power>
        <chanceFactor>0.33</chanceFactor>
        <cooldownTime>1.52</cooldownTime>
        <armorPenetrationBlunt>0.500</armorPenetrationBlunt>
        <linkedBodyPartsGroup>Handle</linkedBodyPartsGroup>
      </li>
      <li Class="CombatExtended.ToolCE">
        <label>point</label>
        <capacities>
          <li>Stab</li>
        </capacities>
        <power>18</power>
        <cooldownTime>1</cooldownTime>
        <armorPenetrationBlunt>11.250</armorPenetrationBlunt>
        <armorPenetrationSharp>90</armorPenetrationSharp>
        <linkedBodyPartsGroup>Point</linkedBodyPartsGroup>
      </li>
      <li Class="CombatExtended.ToolCE">
        <label>edge</label>
        <capacities>
          <li>Cut</li>
        </capacities>
        <power>46</power>
        <cooldownTime>0.5</cooldownTime>
        <armorPenetrationBlunt>45</armorPenetrationBlunt>
        <armorPenetrationSharp>80</armorPenetrationSharp>
        <linkedBodyPartsGroup>Edge</linkedBodyPartsGroup>
      </li>
    </tools>```

Reply:
Fixed in 87c6ef1.

* Rebalanced the melee tools to be similar (slightly inferior) to the CE Melee Wraithblade
* Added `chanceFactor` of 0.33 for handle tool